e2e/cypress/integration/messaging/group_message_spec.js

```diff
@@ -79,4 +90,196 @@ describe('Group Messages', () => {
+
+        cy.location().then((loc) => {
+            const channelId = loc.pathname.split('/').slice(-1)[0];
+
+            // * Assert that sidebar displays the group channel
+            cy.get(`#sidebarItem_${channelId}`).should('contain', `${sortedParticipants[0].username}, ${sortedParticipants[1].username}`);
+
+            // * Assert that member count shows next to the channel name
+            cy.get(`#sidebarItem_${channelId} .status`).eq(0).should('contain', '2');
+        });
+    });
+
+    it('MM-T469 - Post an @mention on a group channel', () => {
+        spyNotificationAs('withNotification', 'granted');
+
+        // # Create a group message with two other users
+        const participants = users.slice(0, 2);
+        createGroupMessageWith(participants);
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // # Post a message as a different user
+        cy.getCurrentChannelId().then((channelId) => {
+            cy.apiLogin(participants[0]);
+
+            cy.postMessageAs(participants[0], `@${testUser.username} Hello!!!`, channelId);
+
+            // * Assert that user receives notification
+            cy.wait(TIMEOUTS.HALF_SEC);
+            cy.get('@withNotification').should('have.been.called');
+        });
+    });
+
+    it('MM-T475 - Channel preferences, mute channel', () => {
+        spyNotificationAs('withNotification', 'granted');
+
+        // # Create a group message with two other users
+        const participants = users.slice(0, 2);
+        createGroupMessageWith(participants);
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // # Clicks on Mute Channel
+        cy.get('#channelHeaderDropdownButton button').click().then(() => {
+            cy.get('#channelToggleMuteChannel button').click().then(() => {
+                // * Assert that channel is muted
+                cy.get('#toggleMute').should('be.visible');
+            });
+        });
+
+        // # Post a message as a different user
+        cy.getCurrentChannelId().then((channelId) => {
+            cy.apiLogin(participants[0]);
+            let channelName;
+
+            cy.location().then((loc) => {
+                channelName = loc.pathname.split('/').slice(-1)[0];
+            });
+
+            cy.postMessageAs(participants[0], 'Hello all', channelId).then(() => {
+                cy.apiLogin(testUser);
+                cy.visit(townsquareLink);
+
+                // * Assert that user does not receives a notification
+                cy.wait(TIMEOUTS.HALF_SEC);
+                cy.get('@withNotification').should('not.have.been.called');
+
+                // * Should not have unread mentions indicator.
+
+                cy.get(`#sidebarItem_${channelName}`).
+                    scrollIntoView().
+                    find('#unreadMentions').
+                    should('not.exist');
+            });
+
+            cy.postMessageAs(participants[0], `@${testUser.username} Hello!!!`, channelId).then(() => {
```

Comment:
And here.

Reply:
see https://github.com/mattermost/mattermost-webapp/pull/7181#discussion_r539229657